index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Colin's Website</title>
+	</head>
+	<body>
+		<nav>
+			<a class="navbar" href="#info">About Me</a>
+			<a class="navbar" href="#portfolio">Portfolio</a>
+			<a class="navbar" href="#contact">Contact</a>
+		</nav>
+				<div class="about-me">
```

Comment:
this div should be at the same indentation level as the nav above it (and all its contents should be adjusted accordingly)